lib/src/widgets/detailCardsWidgets/movie_main_info.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class PosterTitle extends StatelessWidget {
+  const PosterTitle({
+    Key? key,
+    required this.posterPath,
+    required this.id,
+    required this.title,
+    required this.releaseDate,
+    required this.voteAverage,
+  }) : super(key: key);
+
+  final String posterPath;
+  final int id;
+  final String title;
+  final String releaseDate;
+  final num voteAverage;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: UiConstants.widgetPadding),
+      child: Row(
+        children: <Widget>[
+          Hero(
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(
+                UiConstants.posterPathBorderRadius,
+              ),
+              child: Image(
+                image: NetworkImage(
+                  posterPath,
+                ),
+                height: UiConstants.posterPathHeight,
+              ),
+            ),
+            tag: id,
```

Comment:
id can be more descriptive name, you can use heroTag or something like that

Reply:
i deleted this hero , i was not using it.